src/ex2/View.js

```diff
@@ -0,0 +1,196 @@
+export default class View {
+  constructor(addItemHandler, deleteItemHandler, getAllItemsHandler) {
+    this.toDoInput = document.querySelector("input");
+    this.listContainer = document.querySelector("ul");
+    this.alertBox = document.querySelector(".alert");
+    this.alertBoxText = document.querySelector(".alert-innet-text");
+    this.addItemHandler = addItemHandler;
+    this.deleteItemHandler = deleteItemHandler;
+    this.getAllItemsHandler = getAllItemsHandler;
+  }
+
+  init() {
+    const aletBoxCloseBtn = document.getElementById("alert-close-btn");
+    aletBoxCloseBtn.addEventListener("click", () => {
+      this.closeAlertBox();
+    });
+
+    const deleteAllBtn = document.getElementById("clear-all");
+    deleteAllBtn.addEventListener("click", () => {
+      this.listContainer.childNodes.forEach((item) => {
+        this.deleteItem(item);
+      });
+    });
+  }
+
+  get _inputText() {
+    return this.toDoInput.value;
+  }
+
+  resetInput() {
+    this.toDoInput.value = "";
+  }
+
+  getElement(selector) {
```

Comment:
`getElementBySelector`, the function name must be understandable!

Reply:
changed 👍